synapse/lib/storm.py

```diff
@@ -318,6 +341,12 @@
             'additionalProperties': True,
             'required': ['form'],
         },
+        'configvartype': {
+            'anyOf': [
+                {'type': 'array', 'items': {'$ref': '#/definitions/configvartype'}},
+                {'enum': list(s_datamodel.Model().types)},
```

Comment:
This gets us the following core types; but doesn't get the rest of our model types.

```
>>> import synapse.datamodel as s_datamodel
>>> list(s_datamodel.Model().types)
['int', 'float', 'range', 'str', 'hex', 'bool', 'time', 'duration', 'ival', 'guid', 'syn:tag:part', 'syn:tag', 'comp', 'loc', 'ndef', 'array', 'edge', 'timeedge', 'data', 'nodeprop', 'hugenum', 'taxon', 'taxonomy', 'velocity']
```

@invisig0th were you intending for things like `inet:url` or `inet:fqdn` to be valid types here?

Reply:
updated to validate types on add